querybook/webapp/components/DataDocLeftSidebar/DataDocLeftSidebar.tsx

```diff
@@ -72,24 +74,31 @@ export const DataDocLeftSidebar: React.FunctionComponent<IProps> = ({
     let contentDOM: React.ReactChild;
     if (contentState === 'contents') {
         contentDOM = (
-            <div className="sidebar-content sidebar-content-contents">
-                <Level className="contents-panel-header">
-                    <IconButton
-                        icon="ArrowLeft"
-                        onClick={() => setContentState('default')}
-                    />
-                    <div className="flex-row">
-                        <span className="mr4">
-                            contents ({TOGGLE_TOC_SHORTCUT})
-                        </span>
-                        <InfoButton layout={['right', 'top']}>
-                            Click to jump to the corresponding cell. Drag cells
-                            to reorder them.
-                        </InfoButton>
-                    </div>
-                </Level>
-                <DataDocContents cells={cells} docId={docId} />
-            </div>
+            <Resizable
+                defaultSize={{
+                    width: '280px',
```

Comment:
we could make it like a google doc sidebar, where the min and max are both capped?